File: Day3-函数与类继承与派生/ocExtendsDemo/ocExtendsDemo/Rectangle.m
//
//  Rectangle.m
//  ocExtendsDemo
//
//  Created by liu on 13-9-8.
//  Copyright (c) 2013年 liu. All rights reserved.
//

#import "Rectangle.h"

@implementation Rectangle


//设置宽度、高度
-(void)setIHigh:(int) h
{
    iHigh = h;
}


-(void)setIWidth:(int) w
{
    iWidth = w;
}

//打印宽度和高度
-(void)showHW
{
    NSLog(@"iHigh = %d , iWidth = %d \n",iHigh,iWidth);
    
     NSLog(@"x = %d, y = %d  \n",iPoint_x,iPoint_y);
}

@end

File: Day3-函数与类继承与派生/cppExtendsDemo/cppExtendsDemo/Rectangle.cpp
//
//  Rectangle.cpp
//  cppExtendsDemo
//
//  Created by liu on 13-9-8.
//  Copyright (c) 2013年 liu. All rights reserved.
//

#include "Rectangle.h"
#include <iostream>
using namespace std;

void Rectangle::setIHigh(int _h)
{
    iHigh = _h;
}

void Rectangle::setIWidth(int _w)
{
    iWidth = _w;
}

void Rectangle::showHW()
{
    cout<<"heigh  = "<<iHigh<<", width ="<<iWidth<<endl;
}






